contrib/babelfishpg_tds/src/backend/tds/tdstypeio.c

```diff
@@ -3772,6 +3855,48 @@ TdsTypeSqlVariantToDatum(StringInfo buf)
 	PG_RETURN_BYTEA_P(result);
 }
 
+/*
+ * dataformat: totalLen(4B) + baseType(1B) + metadatalen(1B) +
+ * encodingLen(5B) + dataLen(2B) + data(dataLen)
+ */
+static int
+TdsSendTypeSqlvariantAsNVarcharHelper(const char *st)
```

Comment:
this can cause confusion with regular nvarchar sender for sql_variant datatype. Please add comment stating why is it needed

Reply:
Most of this code shares common code with `TdsSendTypeSqlvariant` to send string data, please consider if we can move this into some common helper code. It will help maintain code in long term